addon/templates/components/bs4/bs-accordion/title.hbs

```diff
@@ -0,0 +1,8 @@
+{{! template-lint-disable nested-interactive }}
+<div role="tab" class="card-header {{if collapsed "collapsed" "expanded"}}">
```

Comment:
As Ember components (for now, newer glimmer components are better in this regard IMHO) always have their own root DOM element (unless you set `tagName: ''`), this will result in this `<div  class="card-header">` being wrapped in another (useless) `<div>` from the component itself. Better remove this `<div>` form the template and reuse the components element, by using the special component properties like `classNames`, `classNameBindings`, `role` to customize the root element.